### DISCUSSION TITLE
Minor bug in example

### DISCUSSION THREAD

**Discussion by GitRogal** [tectalichq/public-openai-client-php#12] [General] [answered]
Hello! :)
There is a minor bug in example that shows how to use try/catch block when performing request.
To be specific:  $rawResponse  = $handler->getResponse()->getResponse().
It should be:  $handler->getResponse().

![screen](https://user-images.githubusercontent.com/37812191/218732568-681b7aba-8ab4-47db-b23e-3e5c59d2047e.png)

Thank you for your work!
This tool is incredible!

**Comment by thejamescollins** [comment]
Thank you for reporting this error @GitRogal, much appreciated.

That line will be removed entirely from the **Readme** in the next release of this package.

**Comment by thejamescollins** [comment, marked as answer]
Hi there @GitRogal,

The latest release (version 1.3.1 released today) contains a fix for this, as well as some other improvements in the Readme file.

Please see https://github.com/tectalichq/public-openai-client-php/releases/tag/v1.3.1 for details.

Thanks again for reporting this.
